installer/bundle/bundle_skel.sh

```diff
@@ -801,6 +802,16 @@ case "$installMode" in
                     [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
                 fi
             done
+			echo "----- Installing MySQL ruby gem -----"
+			set +e
+			$OMSAGENT_RUBY_DIR/gem install mysql2 2>/dev/null
```

Comment:
Also, do you really want to direct errors to /dev/null? I'd think that, if errors occurred, you would want the customer to be able to see that ...